src/api/controllers/followController.js

```diff
@@ -0,0 +1,129 @@
+import models, { sequelize } from '../models/index';
+import status from '../../helpers/constants/status.codes';
+import errorSender from '../../helpers/error.sender';
+import errorMessage from '../../helpers/constants/error.messages';
+
+const emptyArrayLength = 0;
+
+/**
+ * @class
+ */
+export default class FollowingController {
```

Comment:
@truestbyheart I can't find the comment but I think I left a comment a few days ago about making the controller name and filename the same.